legacy/.php-cs-fixer.php

```diff
@@ -2,6 +2,7 @@
 
 $finder = PhpCsFixer\Finder::create()
     ->in(__DIR__)
+    ->notPath('application/configs/conf.php')
```

Comment:
... This is only used to allow the weird indent I made using comments. Otherwise its difficult to understand where in the schema you are.